httpie/output/formatters/xml.py

```diff
@@ -0,0 +1,36 @@
+from xml.parsers.expat import ExpatError
+
+from defusedxml.common import DefusedXmlException
+from defusedxml.minidom import parseString
```

Comment:
At first, I wanted to move those imports at line 19. @jakubroztocil LMK if you think it is worth doing it.

Reply:
Agree 👍🏻